Sources/Rendering/OpenGL/Texture/index.js

```diff
@@ -364,6 +364,20 @@ function vtkOpenGLTexture(publicAPI, model) {
       );
     }
 
+    if (
+      [
+        model.context.R32F,
+        model.context.RG32F,
+        model.context.RGB32F,
+        model.context.RGBA32F,
+      ].includes(model.internalFormat) &&
+      !model.context.getExtension('OES_texture_float_linear')
```

Comment:
@sankhesh I noticed that we call getExtension several times in the render path. What if we pre-cached all of the used extensions and looked them up using a cache instead of calling getExtension every time?

Reply:
Yep, I have that change in the branch with memory leak fixes. I'll clean it up and create a PR soon